R/trace_calls.R

```diff
@@ -113,15 +113,7 @@ new_counter <- function(src_ref, parent_functions) {
   .counters[[key]]$value <- 0
   .counters[[key]]$srcref <- src_ref
   .counters[[key]]$functions <- parent_functions
-
-  if (isTRUE(getOption("covr.record_tests", FALSE))) {
-    .counters[[key]]$tests <- matrix(
-      numeric(0L),
-      ncol = 3L,
-      # test index; call stack depth of covr:::count; execution order index
-      dimnames = list(c(), c("test", "depth", "i")))
-  }
-
+  if (isTRUE(getOption("covr.record_tests", FALSE))) new_test_counter(key)
```

Comment:
Another example of cleaning up the "vanilla" path. The block that was removed was moved to be handled in `new_test_counter(key)`.